app/serializers/application_serializer.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationSerializer < ActiveModel::Serializer
+  # ActiveModel Serializers seems to have a bug in which it tries to determine a root key
+  # even though we are using the :attributes adapter, which doesn't use a root.
+  # This shows up when one tries to `render(json: collection)` where collection is some collection of
+  # objects that has a valid guessable serializer.
+  # Setting type to empty string is a workaround for this.
```

Comment:
does it make any sense to add to this comment why setting type to "" is a workaround? or if someone needs to know, we'll find it in the docs?

Reply:
Yeah I'm hoping future versions will fix this.